agenda/models.py

```diff
@@ -256,6 +259,25 @@ def save(self, *args, **kwargs):
                 self.enddate, self.endtime
             )
 
+        body = render_to_string(
+            "email/new_event.txt",
+            {
+                "event": self,
+            },
+        )
+
+        # Send an email to the user when the event is created
+        if self.type != "chore" and self._state.adding:
+            EmailMessage(
+                "[Agenda] " + self.item_title,
+                body,
+                settings.DEFAULT_FROM_EMAIL,
+                [settings.MAILINGLIST],
+                ["bcc@example.com"],
+                reply_to=["another@example.com"],
+                headers={"Message-ID": "foo"},
+            ).send()
+
         super().save(*args, **kwargs)
```

Comment:
So if the email sending fails - then the event does not get saved. 

I assume that is what we want ?

Reply:
Let me revisit, I think the emails are an optional enhancement. So logging and continuing is preferable.

Reply:
Addressed: https://github.com/MakerSpaceLeiden/makerspaceleiden-crm/pull/308/commits/d502a972c83762c8b1025f683affc217653da2e0